bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/GJets_bj_LO_MLM/GJets_bbj_DR-0p4/GJets_bbj_DR-0p4_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+import model sm-ckm_no_b_mass
+# Define multiparticle labels
+define p = u c s d b u~ c~ s~ d~ b~ g
+define j = p
+define l+ = e+ mu+ ta+
```

Comment:
can you also remove the lepton and neutrino definitions, unless they are used? (I don't see it)

Reply:
ping

Reply:
@perrozzi Gouranga is on sudden leaves since two weeks. Please wait him to come back and make another change. Sorry for some delay :-)

Reply:
ciao, does that line slows down the execution, can't we keep it? by the way, if someone can modify the pull request, cleaned datacards without that line are here https://github.com/zhangzc11/genproductions/tree/abjj_abbj_noHT/bin/MadGraph5_aMCatNLO/cards/production/13TeV/GJets_bj_LO_MLM

Reply:
Why do you want to keep them? 

And why don't you modify the PR yourself.

Reply:
Updated the PR. Can you please check once again.